analyzer/src/main/java/org/jboss/gm/analyzer/alignment/DummyAlignmentService.java

```diff
@@ -24,7 +24,7 @@ private String getProjectAlignment(Request request) {
             return result;
         }
 
-        final ProjectVersionRef first = request.getDependencies().get(0);
+        final ProjectVersionRef first = request.getDependencies().iterator().next();
```

Comment:
We shouldn't worry about this for now, since in my next PR this will be going away.

Reply:
OK, what will you use instead? I worry about relying on mixing project and dependencies in the same bag…

Reply:
`DummyAlignmentService` is going to be deleted